src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,22 @@
 import React from 'react';
 import './MoviesList.scss';
+import PropTypes from 'prop-types';
+import { MovieCard } from '../MovieCard/MovieCard';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ moviesFromServer }) => (
+  <>
+    {moviesFromServer.map(item => (
+      <MovieCard key={item.imdbId} {...item} />
+    ))}
+  </>
 );
+
+MoviesList.propTypes = {
+  moviesFromServer: PropTypes.arrayOf(PropTypes.shape({
```

Comment:
It's okay, but in bigger projects it's more convenient to create a separate file with MovieCardShape, so that you could reuse it in many components.

Reply:
Ok, I'll remember